core/nuxeo-operation.js

```diff
@@ -330,27 +413,90 @@ import './nuxeo-connection.js';
         });
     }
 
+    /**
+     * Handler to abort bulk operations executed through the BAF
+     */
+    _abort(commandId) {
+      // CHECK - is this enough to abort the previous request?
+      if (this._controller) {
+        this._controller.abort();
+      }
+      // if the operation is aborted, then on next poll we get the correct state
+      return this.$.nx.request().then((request) =>
+        request
+          .path(`bulk/${commandId}/abort`)
+          .execute({ method: 'put' })
+          .then((status) => {
+            if (this._isAborted(status)) {
+              this.dispatchEvent(
+                new CustomEvent('poll-aborted', {
+                  bubbles: true,
+                  composed: true,
+                  detail: status,
+                }),
+              );
+            }
+            return status;
+          })
+          .catch((error) => {
+            this.dispatchEvent(
+              new CustomEvent('poll-error', {
+                bubbles: true,
+                composed: true,
+                detail: error,
+              }),
+            );
+            throw error;
+          }),
+      );
+    }
+
     _isRunning(status) {
       if (status['entity-type'] === 'bulkStatus') {
-        const { state } = status.value;
+        const { state } = status.value ? status.value : status;
```

Comment:
can you please add a comment here to clarify that this `const` supports both endpoints, for future reference?